README.md

```diff
@@ -51,6 +51,9 @@ This work aims to port PyTorch to HammerBlade.
 
       python setup.py develop
 
+- Turn on emulation debug info
+      export HBEMUL_DEBUG=1
+
```

Comment:
not the correct PR ... but we do we have it doc'ed ;)

Reply:
Cool!